atomic_reactor/schemas/config.json

```diff
@@ -661,6 +661,13 @@
           "limit_media_types": {
             "description": "Do not report any media types other than these",
             "allOf": [{"$ref": "#/definitions/media_types"}]
+          },
+          "blacklist_srpms": {
+              "description": "Don't include specified srpms to source container image",
```

Comment:
It would be nice to explicitly say this field hold the SRPM name (not the nvr nor anything else)

Reply:
ok